src/client/scripts/member.js

```diff
@@ -125,28 +125,32 @@ function showAccountInfo() {
     revealElement(element_accountInfo)
 }
 
-function removeAccount() {
-    if (confirm("Are you sure you want to delete your account? This CAN NOT be undone!")) {
+async function removeAccount(confirmation) {
+    if (!confirmation || confirm("Are you sure you want to delete your account? This CANNOT be undone! Click OK to enter your password.")) {
+        const password = prompt("Enter your password to PERMANENTLY delete your account: ");
         const config = { // Send with our access token
             method: 'DELETE',
             headers: {
                 'Content-Type': 'application/json',
                 'Authorization': `Bearer ${token}`
             },
+			body: JSON.stringify({ password }),
```

Comment:
Add the `username` property in the body!

Reply:
But that's redundant as the username is determined by the Bearer token.